message/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+class Message(models.Model):
+
+    author = models.CharField(max_length=200)
```

Comment:
I think its better to use enum of DogOwner/DayCare so we won't duplicate names, also what will happen if both have same name?

Reply:
in that case how the daycare will know who is the dog owner that sending him a messege? (and the same for the dog owner).
maybe it should inclue the enum that jenia mentioned and concat with the name of the dogOwner/Daycare
for exmaple: daycare.puppies

Reply:
Done, added enum for author